cmake/projects/occt/hunter.cmake

```diff
@@ -0,0 +1,33 @@
+# Copyright (c) 2016-2020, Rahul Sheth, Ruslan Baratov
+# All rights reserved.
+
+# !!! DO NOT PLACE HEADER GUARDS HERE !!!
+
+include(hunter_add_version)
+include(hunter_cacheable)
+include(hunter_download)
+include(hunter_pick_scheme)
+include(hunter_cmake_args)
+
+hunter_add_version(
+    PACKAGE_NAME
+    occt
+    VERSION
+    7.4.0-p1
+    URL
+    "https://github.com/cpp-pm/OCCT/archive/hunter-7.4.0-p1.tar.gz"
```

Comment:
You aren't supposed to create random branches for packages. The correct form is:
hunter-<version>-<patch version>
I released `v7.4.0-p0` from the branch `hunter-7.4.0`, which I merged your incorrectly named `hunter-7.4.0-p1` named branch into. I deleted the incorrect branch.

Please use this release here: https://github.com/cpp-pm/OCCT/releases/tag/v7.4.0-p0